Bottom: af4afd3216b9addcb5809a68e568b3213b6c03a1
Top:    e9931ef7ac5a21466a35ef421d8c12582eb09c1a
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 09:57:34 +0800

debug.patch


---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index c1fc528..aa7ae07 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -881,7 +881,8 @@ static int validate_location(unsigned int logopt, char *loc)
 		    !strncmp(ptr, "file:", 5) || !strncmp(ptr, "yp:", 3) ||
 		    !strncmp(ptr, "nis:", 4) || !strncmp(ptr, "nisplus:", 8) ||
 		    !strncmp(ptr, "ldap:", 5) || !strncmp(ptr, "ldaps:", 6) ||
-		    !strncmp(ptr, "sss:", 4) || !strncmp(ptr, "dir:", 4))
+		    !strncmp(ptr, "sss:", 4) || !strncmp(ptr, "dir:", 4) ||
+		    !strncmp(ptr, "hosts:", 4))
 			return 1;
 		error(logopt,
 		      "expected colon delimeter not found in location %s",
